packages/nc-gui/components/smartsheet/grid/GroupBy.vue

```diff
@@ -154,18 +153,21 @@ const parseKey = (group: Group) => {
   }
 
   // show the groupBy dateTime field title format as like cell format
-  if (key && group.column?.uidt === UITypes.DateTime && dayjs(key).isValid()) {
-    const dateFormat = parseProp(group.column?.meta)?.date_format ?? dateFormats[0]
-    const timeFormat = parseProp(group.column?.meta)?.time_format ?? timeFormats[0]
-
-    const dateTimeFormat = `${dateFormat} ${timeFormat}`
-
-    return [dayjs(key).utc().local().format(dateTimeFormat)]
+  if (key && group.column?.uidt === UITypes.DateTime) {
+    const dateTime = parseStringDateTime(
+      key,
+      `${parseProp(group.column?.meta)?.date_format ?? dateFormats[0]} ${
+        parseProp(group.column?.meta)?.time_format ?? timeFormats[0]
+      }`,
+    )
+
+    return [dateTime]
   }
 
   // show the groupBy time field title format as like cell format
-  if (key && group.column?.uidt === UITypes.Time && dayjs(key).isValid()) {
-    return [dayjs(key).format(timeFormats[0])]
+  if (key && group.column?.uidt === UITypes.Time) {
+    const time = parseStringDateTime(key, timeFormats[0], false)
+    return [time]
```

Comment:
same as above

Reply:
Okay, I will remove this and from other place `Links` -> `ListItem` and related computed values `displayValueTypeAndFormatProp` & `headerDisplayValue` from `useLTARStore.ts` file

Reply:
done